packages/discovery-styles/scss/components/document-preview/_simple-document.scss

```diff
@@ -20,3 +20,33 @@
 .#{$prefix}--simple-document__content {
   max-width: 78ch;
 }
+
+$width: 500px;
+$icon-size: 80px;
```

Comment:
I think it would be helpful for readability to define these at the top of the file, and this is also consistent with `_search-results.scss`, for example